app/controllers/application_controller.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::Base
 
 
   def current_user
-    @current_user ||= Teacher.find(session[:user_id]) if session[:user_id]
+    @current_user ||= Teacher.find(cookies.signed[:user_id]) if cookies.signed[:user_id]
```

Comment:
I don't understand why you did this, but I'm sure it's with good reason. Can you explain to me what the difference is?

Reply:
The reason is the the `cable` cannot access the session